### ISSUE TITLE
Create a PR Template

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/universal-application-tool#33]
Should include "Fixes #..., fixes #...." and "Blocked by #..., #..."

**Comment by neel-a-mehta** [comment]
Include "done when" criteria (eg this is done when X decision is reached, Y bug is fixed...)

**Comment by natsid** [comment]
I think the "done when" part should be part of an issue template (if we create one) rather than the PR template.

**Comment by neel-a-mehta** [comment]
Yes, that sounds right @natsid. Will make another Issue for an Issue template...

**Comment by neel-a-mehta** [comment]
https://docs.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository
